blocks/newslist/newslist.css

```diff
@@ -40,17 +40,29 @@
   font-family: var(--header-font-family);
   font-size: var(--body-font-size-xxs);
   color: #000;
-  margin-bottom: 20px;
   padding: 0 0 20px;
-  line-height: 1.83;
+  line-height: 1.6;
+}
+
+.newslist .newslist-container .newslist-item .newslist-item-description p {
+  margin-bottom: 0;
+}
+
+@media (min-width: 768px) {
```

Comment:
@veejayricana `@media (min-width: 768px) {` is already defined in line 294, can you please move these styles there, rather than defining this break point again?

Reply:
hi @ravkiran, done transferring this code. sorry I think, my branch is not update from the commit since yesterday and I didn't see the other @media (min-width: 768px) {.